submissions/peterIyiola/JS/index2.js

```diff
@@ -0,0 +1,12 @@
+for (let i = 1; i <= 116; i++) 
+{
+    
+if( i % 3 ==0)   
+    console.log("software");
+
+if(i % 5 ==0)
+    console.log("Developer");
+
+if (i % 3 ==0 && i % 5 ==0)
```

Comment:
This should be executed first. Also, make use of `if else if and else` statement